values.yaml

```diff
@@ -213,6 +213,7 @@ enketo:
   env:
     normal: 
       ENKETO_APP_NAME: "Enketo Express for KoboToolbox"
+      ENKETO_MAX_PROCESSES: "2"
```

Comment:
seems intuitively slightly low to me but i guess the container overhead is small enough that it's helpful to facilitate granular scaling. in any case, i defer to your judgment :+1:

Reply:
I'll think about how to review this internally and come up with a default on a new PR. We do need to set this, otherwise when we change node size, it jumps and hits the resource limit/request in an unexpected way that might just OOM crash or screw up scaling.